src/main/java/pl/javastart/sellegro/auction/AuctionRepository.java

```diff
@@ -0,0 +1,31 @@
+package pl.javastart.sellegro.auction;
+
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import java.util.List;
+
+
+@Repository
+public interface AuctionRepository extends JpaRepository<Auction, Long> {
+
+    List<Auction> findAllBy();
+
+    @Query(value="SELECT * FROM auction ORDER BY price DESC LIMIT 4", nativeQuery = true)
```

Comment:
Raczej staramy się korzystać z JPQL a nie z query natywnyh z racji tego, że przy starcie aplikacji zostają sprawdzone pola klasy (czy istnieją itd.). No i alternatywnie można to było zrobić metodą spring data.